tools/go.mod

```diff
@@ -0,0 +1,5 @@
+module tools
+
+go 1.15
+
+require github.com/bflad/tfproviderdocs v0.8.0 // indirect
```

Comment:
**suggestion:** Could we make those generated docs look the same as AWS ones? Also I see you use here `bflad/tfproviderdocs` but in docs you have:  `# generated by https://github.com/hashicorp/terraform-plugin-docs`.

Reply:
This package is indeed wrongly added, I corrected that. Regarding look of the docs I think AWS is using some magic and I wanted to implement docs ASAP and following terraform guides. Maybe in the future they will improve feel of the generated docs.